css/properties/text-underline-offset.json

```diff
@@ -4,9 +4,11 @@
       "text-underline-offset": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-underline-offset",
+          "spec_url": "https://www.w3.org/TR/css-text-decor-4/#underline-offset",
```

Comment:
```suggestion
```

Reply:
We don’t yet add `spec_url` values to BCD for any features other than those in the `javascript` tree. (In https://github.com/w3c/browser-compat-data I have the spec URL data for all existing features, and I personally wish we would go ahead and upstream that to add them everywhere, but we’re not there yet…)

Reply:
@sideshowbarker do we have an issue for `spec_url` everyhwere already? (I looked and couldn't find one.) This _seems like_ something that would be low-risk and high-reward, so please direct me to that issue or open it, so we can consider this more directly. 👍

Reply:
> @sideshowbarker do we have an issue for `spec_url` everywhere already? (I looked and couldn't find one.) This _seems like_ something that would be low-risk and high-reward, so please direct me to that issue or open it, so we can consider this more directly.

I realize now that I hadn’t yet opened a new issue for it. I had originally opened #2983 as a patch for adding spec URLs everywhere, but around https://github.com/mdn/browser-compat-data/pull/2983#issuecomment-430946543 (two years ago now…) the discussion about it with @Elchi3 resolved into deciding to just to it for JavaScript features. And so at that point, I pared the patch down to just that, and also changed the PR title and description to reflect that.

Anyway, I’ve now opened https://github.com/mdn/browser-compat-data/issues/6765 to track the broader plan.

Reply:
So should I revert this and only make a PR with the note?

Reply:
> So should I revert this and only make a PR with the note?

There’s a **Commit suggestion** button above that you can push to apply the suggested commit (which will have the effect of reverting that part of the patch). And yes you should do that — just for consistency with the rest data in the `css` tree, none of which so far as `spec_url` values